Tasks/Common/AzureRmDeploy-common/operations/ContainerBasedDeploymentUtility.ts

```diff
@@ -44,9 +44,14 @@ export class ContainerBasedDeploymentUtility {
             'value': startupCommand
         }
 
+        appSettingsNewProperties.windowsFxVersion = {
```

Comment:
This value will be ignored by ARM call for Linux app and linuxFxVersion will be ignored for Windows app.

Reply:
changed logic